### PR TITLE
Create ui-c8y-1020-5-0-new-alarm-event-selector-can-now-be-used-in-custom-applications.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-59428](https://cumulocity.atlassian.net/browse/MTM-59428)
Original PR: [6641](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6641)

### CODE REVIEW

**Review by eniosultan — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-5-0-new-alarm-event-selector-can-now-be-used-in-custom-applications.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59428
+version: 1020.5.0
+---
+To enable application developers to more easily select alarms and events in their custom applications, a new reusable alarm and event selector component has been introduced.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-5-0-new-alarm-event-selector-can-now-be-used-in-custom-applications.md:17]: @eniosultan Is this described in the Codex? Then I would at least generically add "For details refer to the [{{< product-c8y-iot >}} Codex](https://styleguide.cumulocity.com/apps/codex/#/).

Comment (eniosultan) [inline content/change-logs/application-enablement/ui-c8y-1020-5-0-new-alarm-event-selector-can-now-be-used-in-custom-applications.md:17]:
```suggestion
To enable application developers to more easily select alarms and events in their custom applications, a new reusable alarm and event selector component has been introduced. For details refer to the {{< product-c8y-iot >}} Codex
```

File: content/change-logs/application-enablement/ui-c8y-1020-5-0-new-alarm-event-selector-can-now-be-used-in-custom-applications.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-59428
+version: 1020.5.0
+---
+To enable application developers to more easily select alarms and events in their custom applications, a new reusable alarm and event selector component has been introduced. For details refer to the {{< product-c8y-iot >}} Codex
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-5-0-new-alarm-event-selector-can-now-be-used-in-custom-applications.md:17]:
```suggestion
To enable application developers to more easily select alarms and events in their custom applications, a new reusable alarm and event selector component has been introduced. For details refer to the [{{< product-c8y-iot >}} Codex](https://styleguide.cumulocity.com/apps/codex/#/).
```

**Review by BeateRixen — APPROVED** [review]

**Comment by BeateRixen** [comment]
@rahultsag Please approve.

**Review by rahultsag — APPROVED** [review]

**Pull request merged**